test/inductor/test_distributed_patterns.py

```diff
@@ -118,6 +120,32 @@ def steps(m, inp):
     return out
 
 
+fw_graph = [None]
+bw_graph = [None]
+
+
+def aot_graph_capture_backend(gm, args):
```

Comment:
nit: maybe make this a class so 'fw_graph' can be an instance member instead of a global?